hadoop-project/pom.xml

```diff
@@ -2284,9 +2284,9 @@
               <source>${javac.version}</source>
               <target>${javac.version}</target>
               <compilerArguments>
```

Comment:
`<compilerArgs>` can be used instead of deprecated `<compilerArguments>`
https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#compilerArgs